docs/csharp/programming-guide/classes-and-structs/named-and-optional-arguments.md

```diff
@@ -70,13 +70,14 @@ The following code implements the examples from this section along with some add
 
 ## Optional arguments
 
-The definition of a method, constructor, indexer, or delegate can specify its parameters are required or optional. Any call must provide arguments for all required parameters, but can omit arguments for optional parameters.
+The definition of a method, constructor, indexer, or delegate can specify its parameters are required or optional. Any call must provide arguments for all required parameters, but can omit arguments for optional parameters. Additionally, nullable reference types (`T?`) implicitly have `null` as their default value.
 
 Each optional parameter has a default value as part of its definition. If no argument is sent for that parameter, the default value is used. A default value must be one of the following types of expressions:
   
 - a constant expression;  
 - an expression of the form `new ValType()`, where `ValType` is a value type, such as an [enum](../../language-reference/builtin-types/enum.md) or a [struct](../../language-reference/builtin-types/struct.md);
 - an expression of the form [default(ValType)](../../language-reference/operators/default.md),  where `ValType` is a value type.
+- For nullable value types or nullable reference types (`T?`), the default value is always `null`.
```

Comment:
This isn't correct. All of these declarations are legal. They conform to the first three bullet points:

```csharp
public struct S
{
}

public class C {
    void M(int? size = 5){} // a constant
    void M(string? msg = "Hello World"){} // constant
    void M(S someStruct = new S()){} // new expression, assuming `S` is a `struct`
    void M(object? o = default) { } // OK default expression
    void M(int size, S someStruct = default) {} // OK default expression addition parameter needed
    void M(S? someStruct = default) {} // OK default expression
}